pkg/restore/client.go

```diff
@@ -104,8 +107,18 @@ func (rc *Client) InitBackupMeta(backupMeta *backup.BackupMeta, backend *backup.
 	if err != nil {
 		return errors.Trace(err)
 	}
+	var ddlJobs []*model.Job
+	err = json.Unmarshal(backupMeta.GetDdls(), &ddlJobs)
+	if err != nil {
+		return errors.Trace(err)
+	}
 	rc.databases = databases
+	sort.Slice(ddlJobs, func(i, j int) bool {
+		return ddlJobs[i].BinlogInfo.SchemaVersion < ddlJobs[j].BinlogInfo.SchemaVersion
+	})
```

Comment:
Seems it can be removed, DDLs are re-sorted in other place.

https://github.com/pingcap/br/pull/155/files#diff-2171ce04d98f84f776b0d7e12af33bf4R235-R237
https://github.com/pingcap/br/pull/155/files#diff-e17286a2d1913c042d4769d41babbe88R217-R219